indico/modules/events/contributions/operations.py

```diff
@@ -181,3 +182,22 @@ def create_contribution_from_abstract(abstract, contrib_session=None):
                                        'session': contrib_session,
                                        'person_link_data': {link: True for link in contrib_person_links}},
                                custom_fields_data=custom_fields_data)
+
+
+def clone_contribution(event, contribution, preserve_session):
+    shared_data = {
+        'event_persons': {'person_map': {person: person for person in event.persons}},
```

Comment:
You could actually do this: `dict(zip(event.persons, event.persons))`

The literals you used are also quite clear, just mentioning that this would work too :p